### PR TITLE
Hiv cbs indicator fields

### PR DESCRIPTION
**Pull request by mariusnaude**
@Lumiere-Lodi  PR created for your review

### CODE REVIEW

File: input/fsh/hiv-vocabulary.fsh [hunk 1]
```diff
@@ -111,3 +111,59 @@ Description: ""
...
+* #RESULTS-NOT-AVAILABLE "Results not available"
+* #ROUTINE-VIRAL-LOAD "Routine Viral Load"
+
+ValueSet: VSHIVVLReason
+Id: vs-hiv-vl-reason
+Title: "HIV VL Reason codes"
+Description: "Valueset - Viral Load test reasons"
+* include codes from system CSHIVVLReason
+
+CodeSystem: CSARTStatus
```
Comment (Lumiere-Lodi) [inline input/fsh/hiv-vocabulary.fsh:138]: Same above. We cant use snomed or icd or loinc?

File: input/fsh/hiv-case-record.fsh [hunk 2]
```diff
@@ -245,6 +260,7 @@ Description: ""
 * identifier.system = "http://openhie.org/fhir/hiv-casereporting/identifier/enrollment-unique-id"
 * managingOrganization 1..1 MS
 * diagnosis 1..1 MS
+* referralRequest MS  //Transfer-Out
```
Comment (Lumiere-Lodi) [inline input/fsh/hiv-case-record.fsh:263]: Shouldnt this field be mandatory? referralRequest 1..1 ?

**Review by Lumiere-Lodi — COMMENTED** [review]
LGTM
Minor comments just to double check before we merge in.

File: input/fsh/hiv-case-record.fsh [hunk 3]
```diff
@@ -50,19 +50,31 @@ Title: "HIV Case Reporting Composition"
 
 * section[hivEntryToCareSection].title = "HIV Entry To Care"
```
[the remark this answers is not in the document]
Comment (mariusnaude) [reply input/fsh/hiv-case-record.fsh:51]: Ok, changed type to "pattern" - I hope that is the only change?

[the remark this answers is not in the document]
Comment (mariusnaude) [reply input/fsh/hiv-case-record.fsh:51]: Yes:

File: input/fsh/hiv-vocabulary.fsh [hunk 4]
```diff
@@ -111,3 +111,59 @@ Description: ""
 * #HIV-ACTUAL-RECENCY-TEST-RESULT "HIV actual recency test result" 
 * #VL-MOST-RECENT-TEST-DATE "VL most recent test date" 
 * #VL-MOST-RECENT-TEST-RESULT "VL most recent test result" 
+
+CodeSystem: CSHIVVLReason
```
[the remark this answers is not in the document]
Comment (mariusnaude) [reply input/fsh/hiv-vocabulary.fsh:115]: We still need to resolve the correct id's to use with our stakeholders for HIV. As far as I know, the terminologies for HIV will be resolved during the Y3 work. This is the codes we received, perhaps I should just use this to align with the datasource: 
image

[the remark this answers is not in the document]
Comment (mariusnaude) [reply input/fsh/hiv-vocabulary.fsh:115]: Agree ?

File: input/fsh/hiv-vocabulary.fsh (hunk 1 above)
Comment (mariusnaude) [reply input/fsh/hiv-vocabulary.fsh:138]: We have not received the terminology codes, only the UUID's

File: input/fsh/hiv-case-record.fsh (hunk 2 above)
Comment (mariusnaude) [reply input/fsh/hiv-case-record.fsh:263]: Why would every HIVEpisodeOfCare have a transfer out request? We did request the field cardinalities as well, but I'm not sure about making this mandatory.

File: input/fsh/hiv-vocabulary.fsh (hunk 1 above)
Comment (mariusnaude) [reply input/fsh/hiv-vocabulary.fsh:138]: @Lumiere-Lodi please check again, changes committed

**Comment by mariusnaude** [comment]
@Lumiere-Lodi  Just need to clarify the following for the ServiceRequest and Procedure recently added. I created an example for both.

You will see that not all profiles are added to the Composition/BundleExample/CompositionExample. For example refer to ViralLoadSuppressionMostRecentTestDate

Should the following  be added to Composition/BundleExample/CompositionExample structures:

- ServiceRequest:  HIVTransferOut already added to these sections - is this correct?
- Procedure: Should this be added to these sections?

**Pull request merged**